templates/deno/remix.config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   serverBuildTarget: "deno",
   server: "./server.ts",
-
-  /*
-  If live reload causes page to re-render without changes (live reload is too fast),
-  increase the dev server broadcast delay.
-
-  If live reload seems slow, try to decrease the dev server broadcast delay.
-  */
+  // If live reload causes page to re-render without changes (live reload is too fast),
+  // increase the dev server broadcast delay.
+  // If live reload seems slow, try to decrease the dev server broadcast delay.
   devServerBroadcastDelay: 300,
+  ignoredRouteFiles: ["**/.*"],
```

Comment:
Why is `ignoredRouteFiles` added?

Reply:
We have these in all other templates as well

Reply:
I don't think we should aim to make this consistent across templates. It's not needed by this template. If it's common enough that we'd want it in every template, we should just make it the default value instead of including it here.

Reply:
All templates have set `ignoredRouteFiles` like this, so they all have a sensible default value.
People could still choose to remove it if they want to.

Same reason those other commented values are there.

If we want to remove it, I think it should be done for all templates consistently.

So let's merge these changes so all templates are consistent & we can remove it again in another PR if we want to.

Reply:
I have similar thoughts about this. If we can directly enunciate the purpose of setting `ignoredRouteFiles`, I'm ok to keep it. But I don't want to include it just because other templates do.

Let's instead keep as-is and make another PR to fix in other templates.

Reply:
The value we set in `ignoredRouteFiles` by default is to comply to the case where `.DS_Store` was causing errors.
That's why we set all dot files to be ignored as the sensible default for an app, as most people will want that.

Reply:
Ok I buy that. Let's keep your `ignoredRouteFiles` changes in the PR.

Reply:
Than I think we can merge this PR.